src/assets/wise5/components/common/feedbackRule/edit-feedback-rules/edit-feedback-rules.component.ts

```diff
@@ -54,12 +54,12 @@ export class EditFeedbackRulesComponent implements OnInit {
     this.projectService.nodeChanged();
   }
 
-  addNewRule(position: string): void {
+  addNewRule(position: number): void {
     const newFeedbackRule = this.createNewFeedbackRule();
-    if (position === 'beginning') {
+    if (position < 0) {
       this.feedbackRules.unshift(newFeedbackRule);
     } else {
-      this.feedbackRules.push(newFeedbackRule);
+      this.feedbackRules.splice(position, 0, newFeedbackRule);
     }
     this.projectService.nodeChanged();
```

Comment:
Can we change this function body to:
```
    this.feedbackRules.splice(position, 0, this.createNewFeedbackRule());
    this.projectService.nodeChanged();
```
and change calls to ```addNewRule(-1)``` to ```addNewRule(0)```?

Reply:
Ah, yes. All set.